framework/db/Migration.php

```diff
@@ -552,9 +538,36 @@ public function dropCommentFromColumn($table, $column)
      */
     public function dropCommentFromTable($table)
     {
-        echo "    > drop comment from table $table ...";
-        $time = microtime(true);
+        $time = $this->beginCommand("drop comment from table $table");
         $this->db->createCommand()->dropCommentFromTable($table)->execute();
-        echo ' done (time: ' . sprintf('%.3f', microtime(true) - $time) . "s)\n";
+        $this->endCommand($time);
+    }
+
+    /**
+     * Prepares for a command to be executed, and outputs to the console
+     * 
+     * @param string $description the description for the command, to be output to the console.
+     * @return float the time before the command is executed, for the time elapsed to be calculated.
+     * @since 2.0.13
+     */
+    protected function beginCommand($description)
```

Comment:
It's not just for printing though. Think of it like beforeSave and afterSave events for example. They're fundamentally different events. It does allow for more flexibility to do any other kind of prep work before actually running the DB operation that whoever extends it might want to do. If we made it just one function you'd probably still need some parameter to differentiate if it's a begin or end to get the same effect which is awkward design I think. I doubt there will be many examples of people extending Migration for changing logging though, so I don't think it's too big a deal to need to extend both if you're one of the few who might want to change the way it's logged.